creatorkit/Glossary.md

```diff
@@ -0,0 +1,19 @@
+![Creator Logo](../creatorlogo.png)
+
+# Glossary
```

Comment:
You can do it wherever you want, but I thought these should be in there as they are quite important terms for people to be aware of